mxnet/gpu/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM rocker/cuda-dev:3.5.2
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    wget cmake && \
+    git clone --recursive https://github.com/dmlc/xgboost && \
+    mkdir -p xgboost/build && cd xgboost/build && \
+    cmake .. -DUSE_CUDA=ON -DR_LIB=ON -DUSE_NCCL=ON && \
+    make install -j$(nproc)
+
+RUN apt-get update && apt-get install -y --no-install-recommends --no-upgrade \
+    libopenblas-dev \
+    liblapack-dev \
+    libopencv-dev \
+    libxt-dev
+    
+RUN git clone --recursive https://github.com/apache/incubator-mxnet.git \
+  &&  cd incubator-mxnet \
+  &&  echo "USE_OPENCV = 1" >> ./config.mk \
+  &&  echo "USE_BLAS = openblas" >> ./config.mk \
+  &&  echo "USE_CUDA = 1" >> ./config.mk \
+  &&  echo "USE_CUDA_PATH = $CUDA_HOME" >> ./config.mk \
+  &&  echo "USE_CUDNN = 1" >> ./config.mk \
+  &&  make -j $(nproc) 
+
+RUN cd incubator-mxnet \
+  && make rpkg
+
+FROM rocker/cuda:3.5.2
+
+RUN apt-get update && apt-get install -y --no-install-recommends --no-upgrade \
+    libopenblas-dev \
+    liblapack-dev \
+    libopencv-dev \
+    libxt-dev
+
+COPY --from=0 /usr/local/lib/R/site-library /usr/local/lib/R/site-library
```

Comment:
copying over all of site-lbrary seems a little rough around the edges -- it's just one R package that needs to be copied, yes?  did:

```
COPY --from=0 /usr/local/lib/R/site-library/mxnet /usr/local/lib/R/site-library/mxnet 
```

not work somehow?

Reply:
It's a bit more than that.  There's xgboost, too, and `mxnet`'s build script installs all its Imports and Suggests, as well.  17 packages I think.  I tried doing `devtools::install_deps(system.file(".", package="mxnet"), dependencies = TRUE))` in the final container, but this was faster.

Reply:
right, cool, that makes perfect sense